pkg/plugin/standalone/v1/destination.go

```diff
@@ -103,21 +102,6 @@ func (s *destinationPluginClient) Write(ctx context.Context, r record.Record) er
 	return nil
 }
 
-func (s *destinationPluginClient) writeErrorCause(err error) error {
```

Comment:
This was removed because we have a separate node that checks the incoming stream. If an error is returned from the stream it will be caught by `AckerNode`, we don't need to do tricks to return it from the outgoing stream.